### PR TITLE
Bug fixes for online inference and introduce online inference based on yaml

### PR DESCRIPTION
**Pull request by Bourn3z**
2.introduce online inference based on yaml.

Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/det/dbnet/db_r50_icdar15.yaml [hunk 1]
```diff
@@ -158,35 +158,36 @@ eval:
 
 predict:
   ckpt_load_path: tmp_det/best.ckpt
+  output_save_dir: ./output
   dataset_sink_mode: False
   dataset:
     type: PredictDataset
     dataset_root: path/to/dataset_root
-    data_dir: ic15/det/test/ch4_test_images
+    data_dir: images/
```
Comment (panshaowu) [inline configs/det/dbnet/db_r50_icdar15.yaml:166]: 该默认值似可不改，可能用户仍是使用ic15测试集进行评估。

File: configs/rec/crnn/crnn_resnet34.yaml [hunk 2]
```diff
@@ -156,36 +156,29 @@ predict:
   dataset:
     type: PredictDataset
     dataset_root: path/to/dataset_root
-    data_dir: predict_result/crop
+    data_dir: images/
```
Comment (panshaowu) [inline configs/rec/crnn/crnn_resnet34.yaml:159]: 该默认值似可不改，可能用户仍是使用ic15测试集进行评估。

File: configs/rec/crnn/crnn_resnet34.yaml [hunk 3]
```diff
@@ -156,36 +156,29 @@ predict:
...
-#          max_text_len: *max_text_len
-#          character_dict_path: *character_dict_path
-#          use_space_char: *use_space_char
-#          lower: True
-      - RecResizeImg: # different from paddle (paddle converts image from HWC to CHW and rescale to [-1, 1] after resize.
-          image_shape: [32, 100] # H, W
-          infer_mode: *infer_mode
-          character_dict_path: *character_dict_path
-          padding: False # aspect ratio will be preserved if true.
-      - NormalizeImage:  # different from paddle (paddle wrongly normalize BGR image with RGB mean/std from ImageNet for det, and simple rescale to [-1, 1] in rec.
```
Comment (panshaowu) [inline configs/rec/crnn/crnn_resnet34.yaml:177]: 推理阶段不需要进行规范化吗？

File: configs/det/dbnet/db_r50_icdar15.yaml (hunk 1 above)
Comment (Bourn3z) [reply configs/det/dbnet/db_r50_icdar15.yaml:166]: 好的

File: configs/rec/crnn/crnn_resnet34.yaml (hunk 2 above)
Comment (Bourn3z) [reply configs/rec/crnn/crnn_resnet34.yaml:159]: 已改回

File: configs/rec/crnn/crnn_resnet34.yaml (hunk 3 above)
Comment (Bourn3z) [reply configs/rec/crnn/crnn_resnet34.yaml:177]: 默认值没有归一化，且eval也没包含。

**Review by panshaowu — APPROVED** [review]

**Review by chengtianwu — APPROVED** [review]

**Pull request merged**